pkg/controller/certificates/requestmanager/requestmanager_controller.go

```diff
@@ -370,11 +370,12 @@ func (c *controller) createNewCertificateRequest(ctx context.Context, crt *cmapi
 			OwnerReferences: []metav1.OwnerReference{*metav1.NewControllerRef(crt, certificateGvk)},
 		},
 		Spec: cmapi.CertificateRequestSpec{
-			Duration:  crt.Spec.Duration,
-			IssuerRef: crt.Spec.IssuerRef,
-			Request:   csrPEM.Bytes(),
-			IsCA:      crt.Spec.IsCA,
-			Usages:    crt.Spec.Usages,
+			Duration:   crt.Spec.Duration,
```

Comment:
We are not issuing a new certificate when the `MaxPathLen` is changed. I would expect this would be the case as a user.

Reply:
Thinking more, I guess we can't actually do anything here- we shouldn't go into a renew loop if the returned cert from an issuer didn't set the `pathLen`.

Reply:
We should make this clear in the docs for `Certificates` ^

Reply:
will add these docs now :+1: